### PR TITLE
Added Hindi translation of README.md (hindi reviewers requested)

### PR DESCRIPTION
**Pull request by CharithaCS**
## PR Checklist

Please check if your PR fulfills the following requirements:

> Put "x" to check

- [x] The commit message follows our [Contributing Guidelines](https://github.com/fityannugroho/idn-area/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (**optional**, for bug fixes or features)
- [x] Docs have been added / updated (**optional**, for bug fixes or features)

## PR Type

What kind of change does this PR introduce?

> Please check any kind of changes that applies to this PR using "x"

- [ ] Bug fix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] *..... (describe the other type)*

## What is the current behavior?

> Please describe the current behavior that you are modifying, or link to a relevant issue.

Issue Number: #95

## What is the new behavior?

Add hindi translation of README

## Other information

None

### CODE REVIEW

**Comment by fityannugroho** [comment]
Will request hindi reviewer.

**Comment by CharithaCS** [comment]
Alright, Thanks!

File: docs/README_hi.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  <a href="https://www.typescriptlang.org"><img alt="टाइप स्क्रिप्ट" src="https://img.shields.io/badge/-TypeScript-007ACC?style=flat-square&logo=typescript&logoColor=white" /></a>
+  <a href="https://www.prisma.io"><img alt="प्रिज्मा" src="https://img.shields.io/badge/-Prisma-1B222D?style=flat-square&logo=prisma&logoColor=white" /></a>
+  <a href="https://www.mongodb.com"><img alt="मोंगो डी बी" src="https://img.shields.io/badge/-MongoDB-47A248?style=flat-square&logo=mongodb&logoColor=white" /></a>
+  <a href="https://www.postgresql.org"><img alt="पोस्टग्रेएसक्यूएल" src="https://img.shields.io/badge/-PostgreSQL-657991?style=flat-square&logo=postgresql&logoColor=white" /></a>
+  <a href="https://www.mysql.com"><img alt="माई एसक्यूएल" src="https://img.shields.io/badge/-MySQL-00688F?style=flat-square&logo=mysql&logoColor=white" /></a>
+</p>
+
+एपीआई जो इंडोनेशिया के **प्रशासनिक क्षेत्रों**, प्रांत, रीजेंसी, जिले से लेकर ग्राम स्तर तक की जानकारी प्रदान करता है। यह [संस्करण 1.1.0] से द्वीप डेटा भी प्रदान करता है (https://github.com/fityannugroho/idn-area/releases/tag/v1.1.0)।
```
Comment (fityannugroho) [inline docs/README_hi.md]: This link will not rendered well. Can you fix it?

**Review by fityannugroho — CHANGES_REQUESTED** [review]
I only found one thing that needs to be fixed

**Comment by CharithaCS** [comment]
Sure, Thanks for bringing this to my attention.

**Comment by CharithaCS** [comment]
Updated. please check.

**Comment by fityannugroho** [comment]
We just updated the README especially on the section related to API documentation. So please update the translation to be match with the latest README (see #198).

You also need to update your fork branch using [Sync fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to pull the updated workflow so that this PR can pass the checks.

**Comment by CharithaCS** [comment]
Done!

**Review by fityannugroho — APPROVED** [review]

**Pull request merged**